src/lib/UI/Config/Mapper/FieldType/RichText/CustomTag.php

```diff
@@ -11,7 +11,7 @@
 use EzSystems\EzPlatformAdminUi\UI\Config\Mapper\FieldType\RichText\CustomTag\AttributeMapper;
 use RuntimeException;
 use Symfony\Component\Asset\Packages;
-use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\Translation\DataCollectorTranslator as Translator;
```

Comment:
Hey @webhdx, I have a bit of an issue here. We're now using `trans` and `getCatalogue` methods from the Translator, however those are implemented by two separate interfaces `TranslatorInterface` and `TranslatorBagInterface`. I cannot find common interface to type-hint. I'm using `DataCollectorTranslator` because it's autowired by default by the Container.
Is there any other way to fix it or can we keep it that way?

Reply:
It's okay for me. Another solution but an overkill would be to have two separate services injected typed by `TranslatorInterface` and `TranslatorBagInterface`. In reality you will inject the same service twice though. So I'm more convinced to your solution.

Reply:
Anyway, looks like `DataCollectorTranslator` cannot be autowired. You should rather explicitly set the argument in the service definition.

Reply:
> Anyway, looks like `DataCollectorTranslator` cannot be autowired. You should rather explicitly set the argument in the service definition.

Yeah, I wonder why it works for me, even though `debug:autowiring` confirms it shouldn't.

Reply:
Changed to `Translator` from the same namespace, this should get auto-wired correctly.